### PR TITLE
Created a new workflow for available subnets

### PR DESCRIPTION
**Pull request by manjoosh12**
Created a new workflow for available subnets

### CODE REVIEW

**Review by umavanth21 — CHANGES_REQUESTED** [review]
If worker_group is not added, please update

**Comment by manjoosh12** [comment]
> If worker_group is not added, please update

added the worker_group

File: content/io/cloudslang/amazon/aws/vpc/available_subnets.sl [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+    - endpoint:
+        default: 'https://ec2.amazonaws.com'
+        required: false
+    - identity
+    - credential:
+        sensitive: true
+    - proxy_host:
+        required: false
+    - proxy_port:
+        default: '8080'
```
Comment (patmante) [inline content/io/cloudslang/amazon/aws/vpc/available_subnets.sl]: remove the default proxy_port

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/amazon/aws/vpc/available_subnets.sl (hunk 1 above)
Comment (manjoosh12) [reply content/io/cloudslang/amazon/aws/vpc/available_subnets.sl]: removed

**Review by umavanth21 — CHANGES_REQUESTED** [review]
1. Most of the places are  having spelling mistake for "default"
2. One question - two get availability zone- one with _1 and other one . What is the difference?
3. why default port mentioned?

**Review by KalyankarRajesh — APPROVED** [review]

**Comment by manjoosh12** [comment]
i have corrected the spelling mistakes and removed default port

**Comment by umavanth21** [comment]
> 

reg Item 2?

**Comment by manjoosh12** [comment]
> > 
> 
> reg Item 2?

i have re corrected that name as well accordingly

**Review by umavanth21 — APPROVED** [review]

**Review by patmante — APPROVED** [review]

**Pull request merged**